### ISSUE TITLE
Unable to tap on "get your Person credential" (Person Credential Splash Screen)

### ISSUE THREAD

**Issue by cvarjao** [bcgov/bc-wallet-mobile#858]
Unable to click "get your Person credential" when font size is set to large

![IMG_0145](https://user-images.githubusercontent.com/1348549/215215922-21b41822-b3bd-4bfb-88c0-dce78ff0cde8.PNG)

**Comment by knguyenBC** [comment]
Please make all screens scrollable to avoid content behind cut below the fold.
